src/main/kotlin/org/stellar/walletsdk/Auth.kt

```diff
@@ -25,13 +28,13 @@ class Auth(
   data class AuthToken(val token: String)
   data class AuthTransaction(val transaction: String)
 
-  fun authenticate(): String {
+  suspend fun authenticate(): String {
```

Comment:
I notice we are defining functions without classes. However, some popular Kotlin libraries wrap functions with classes and interfaces. One of the example is the [Json.kt](https://github.com/Kotlin/kotlinx.serialization/blob/master/formats/json/commonMain/src/kotlinx/serialization/json/Json.kt) of the [kotlinx.serialization library](https://github.com/Kotlin/kotlinx.serialization)

Reply:
Although functions as the first-class object is great, there are many advantages of wrapping the functions with classes in the design of an SDK.